hw_twitterapi/twitterapiapp/templates/twitterapiapp/main_page.html

```diff
@@ -49,15 +49,26 @@ <h1 class="mt-1">Find Following List of a User <span class="text-muted">(by <a h
             <input class="form-control ml-2" type="text" placeholder="Count" id="followings_count">
             <button class="btn btn-primary ml-2" data-url="" id="followings_send">Send</button>
         </div>        
-        <p>Web interface: <code><a href="http://206.189.12.235/followings/screen_name?count=20" id="followings_web">http://206.189.12.235/followings/screen_name?count=20</a></code></p>
+        <p>Web interface for API call: <code><a href="http://206.189.12.235/followings/screen_name?count=20" id="followings_web">http://206.189.12.235/followings/screen_name?count=20</a></code></p>
         <p>API call: <code><a href="http://206.189.12.235/api/followings/screen_name?count=20" id="followings_api">http://206.189.12.235/api/followings/screen_name?count=20</a></code></p>
         <p>Gets the list of users that the user with given screen name follows. By default, gets last 20 users, however custom number can be set using (optional) 'count' parameter.</p>
 
-        {% comment "Hide non-merged api documentation." %}
+        <hr>
+
+        <h1 class="mt-1">Recent Favorite Statistics of a User<span class="text-muted">(by <a href="https://github.com/mfatihm">mfatihm</a>)</span></h1>
+        <div class="form-inline my-2">
+            <input class="form-control" type="text" placeholder="Username" id="recent_favorites_username">
+            <button class="btn btn-primary ml-2" data-url="" id="recent_favorites_send">Send</button>
+        </div>
+        <p>Web interface for API call: <code><a href="{% url 'twitterapiapp:recent_favorites_api' 'issizPecorin'%}">{% url 'twitterapiapp:recent_favorites_api' 'issizPecorin'%}</a></code></p>
+        <p>API call: <code><a href="{% url 'twitterapiapp:recent_favorites' 'issizPecorin'%}">{% url 'twitterapiapp:recent_favorites' 'issizPecorin'%}</a></code></p>
```

Comment:
An `id` attribute should be set in `a` tag. Otherwise it's not reachable by Javascript code and consider using an id compatible with conventions, refer to other parts.